### ISSUE TITLE
display comment in pre tag

### ISSUE THREAD

**Issue by SimonLab** [emfoundation/ce100-app#1028]
Instsad of using p tag for comments:
https://github.com/emfoundation/ce100-app/blob/12c0a085444bc9d94ffbe2188dfdb73b1e498a05/templates/partials/comments/comment.html#L79

we can use the ```pre``` tag. This will display the comment with the original spaces and indentation when the comments were created

The spaces of the comment will be display correctly like the following:
![image](https://user-images.githubusercontent.com/6057298/32316813-a2296590-bfa9-11e7-8500-ca8d8d24e6b2.png)

**Comment by asimonw** [comment]
@SimonLab Works well! Nice idea to use pre tags.
@iteles A nice to have would be to trim extra white space before and after the text. As it is now, a user can add many empty newlines for no good reason. This was not specified in the original scope and is not high priority. But if it's a quick fix and you have some time left this sprint, by all means 😄

**Comment by SimonLab** [comment]
@asimonw the empty spaces should be removed now when a comment is added or edited

**Comment by asimonw** [comment]
@SimonLab @Cleop I think there's something a bit misaligned between the current state and the "undo" state while editing. When I add extra lines at the start or end, the comment adapts well on save, but going back into edit makes the lines reappear. 

I noticed that on Cancel, it did seem to remove the lines correctly, but then realised that it also happens to actual text. I logged this in what I think is the simplest case in #1047
